target/linux/mvebu/image/Makefile

```diff
@@ -212,4 +225,17 @@ define Device/globalscale-mirabox
 endef
 TARGET_DEVICES += globalscale-mirabox
 
+define Device/turris-omnia
+  $(Device/NAND-256K)
+  KERNEL_INSTALL := 1
+  KERNEL := dtb | kernel-bin
+  DEVICE_DTS := armada-385-turris-omnia
+  DEVICE_TITLE := Turris Omnia
+  DEVICE_PACKAGES := swconfig wpad-mini kmod-ath9k kmod-ath10k ath10k-firmware-qca988x
```

Comment:
Maybe @oerdnj can give some numbers of how many Omnia shipped without wifi so we have some data added to the discussion.